jest.config.js

```diff
@@ -115,9 +116,7 @@ module.exports = {
   // rootDir: undefined,
 
   // A list of paths to directories that Jest should use to search for files in
-  // roots: [
-  //   "<rootDir>"
-  // ],
+  roots: ['<rootDir>/dist'],
```

Comment:
I think when we use `roots` we can remove the `testPathIgnorePatterns` config.

Reply:
Discovered very well